src/server.h

```diff
@@ -1281,6 +1280,51 @@ typedef struct redisTLSContextConfig {
     int session_cache_timeout;
 } redisTLSContextConfig;
 
+/*-----------------------------------------------------------------------------
+ * AOF meta definition
+ *----------------------------------------------------------------------------*/
+
+/* Naming rules */
+#define BASE_AOF_SUFFIX      "_b_"       /* server.aof_filename_b_1、server.aof_filename_b_2 etc. */
+#define INCR_AOF_SUFFIX      "_i_"       /* server.aof_filename_i_1、server.aof_filename_i_2 etc. */
+#define META_NAME_SUFFIX     "_manifest" /* server.aof_filename_manifest. */
+#define META_TEM_NAME_PREFIX "temp_"     /* temp_server.aof_filename_manifest. */
+
+/* AOF meta key */
+#define AOF_META_KEY_FILE_NAME   "fileName"
+#define AOF_META_KEY_FILE_SEQ    "fileSeq"
+#define AOF_META_KEY_FILE_TYPE   "fileType"
+
+#define AOF_INFO_FORMAT(info)                               \
+    sdscatprintf(buf,"%s %s %s %" PRId64 " %s %c\n",        \
+                AOF_META_KEY_FILE_NAME,info->file_name,     \
+                AOF_META_KEY_FILE_SEQ,info->file_seq,       \
+                AOF_META_KEY_FILE_TYPE,info->file_type)     \
```

Comment:
in the past, we attempted to avoid using PRId64 and instead just used `long long` (or `long long` casting).

p.s. why is this in server.h? i presume that aof.c is the only one that needs to be aware of these details (including some of the defines above).